src/duckdb/creator.rs

```diff
@@ -418,6 +423,8 @@ impl TableManager {
         // DuckDB doesn't add IF NOT EXISTS to CREATE TABLE statements, so we add it here.
         let create_stmt = create_stmt.replace("CREATE TABLE", "CREATE TABLE IF NOT EXISTS");
 
+        tracing::trace!("{create_stmt}");
```

Comment:
Probably should be more descriptive.

Reply:
Decided to remove this tracing at all, thank you